roles/rsyslog/README.md

```diff
@@ -213,19 +213,24 @@ Common sub-variables
 
 Elasticsearch, Files and Forwards outputs sub-variables
 ----------------------------------
-- `elasticsearch`: array of dictionary to specify the parameters to forward the log messages to elasticsearch.
+- `use_local_omelasticsearch_cert` : Global variable. If set to 'true', local files ca_cert_src, cert_src and key_src in the elasticsearch type logging_outputs will be deployed to the remote host.
```

Comment:
`logging_use_local_elasticsearch_cert` - variable names must have a `logging_` prefix, and I don't think you need the `om` here.

Reply:
What is a "sub-variable" anyway? I don't see it defined anywhere.
If a key in some dictionary is meant, then it does not need a `logging_` prefix (dictionary keys do not need to be namespaced, that would lead to very long and unreadable names.) But "Global variable" indicates that it is not the case.

Reply:
> `logging_use_local_elasticsearch_cert` - variable names must have a `logging_` prefix, and I don't think you need the `om` here.

Ahhh, sorry, @richm. I'm planning to eliminate use_local_omelasticsearch_cert. Instead, using a cert or not is specified in each elasticsearch setting by `use_cert` (see the following example). @pcahyna, what'd be the good terminology to describe `use_cert` or that level of keys?
```
        logging_outputs:
          - name: elasticsearch_output
            type: elasticsearch
            server_host: logging-es
            server_port: 9200
            index_prefix: project.
            input_type: ovirt
            retryfailures: off
            use_cert: false
          - name: elasticsearch_output_ops
            type: elasticsearch
            server_host: logging-es-ops
            server_port: 9200
            index_prefix: .operations.
            input_type: viaq
            retryfailures: off
            use_cert: false
```

Reply:
@nhosoi  Sorry for the late reply. In the network role, we call them "options" https://github.com/linux-system-roles/network/blob/master/README.md#options and I think this is the best option - also used in networking modules, see here: https://docs.ansible.com/ansible/latest/modules/junos_l3_interfaces_module.html#parameter-config. But the terminology can be cleaned up later.

Reply:
@pcahyna, Thank you for the right word. I agree I should change the terminology later with more correction candidates. Also, I should restructure the docs better. I'm opening a RHELPLAN for that. Thanks! > @richm